plugins/plugin-java/che-plugin-java-ext-lang-server/src/main/java/org/eclipse/che/plugin/java/server/rest/recommend/similarity/LevenshteinDistance.java

```diff
@@ -0,0 +1,24 @@
+package org.eclipse.che.plugin.java.server.rest.recommend.similarity;
```

Comment:
Is this class https://github.com/apache/lucene-solr/blob/releases/lucene-solr/7.0.1/lucene/suggest/src/java/org/apache/lucene/search/spell/LevensteinDistance.java doing what you want?
I recommend to use this artifact http://search.maven.org/#artifactdetails%7Corg.apache.lucene%7Clucene-suggest%7C7.0.1%7Cjar instead of writing a custom one.